content/en/docs/tasks/tools/install-kubectl-macos.md

```diff
@@ -287,6 +287,35 @@ Below are the procedures to set up autocompletion for Bash, Fish, and Zsh.
    rm kubectl-convert kubectl-convert.sha256
    ```
 
+### Uninstall kubectl on macOS
+
+Depending on how you installed `kubectl`, use one of the following methods.
+
+### Uninstall kubectl using the command-line
+
+1. Locate the `kubectl` binary on your system:
+
+```bash
+where kubectl
+```
+
+2. Remove the `kubectl` binary:
```

Comment:
Nit, make the numbering `1.` for every item, the markdown processor will figure out the actual numbers

Reply:
ohk. I will make the change